.github/workflows/ci-server.yaml

```diff
@@ -18,7 +18,7 @@ concurrency:
 
 jobs:
   server-test:
-    runs-on: ubuntu-latest
+    runs-on: ci-8-cores
```

Comment:
is it possible to leave it on ubuntu-latest? as we pay for ci-8-cores we only use it when we see drastic ci duration changes